mg/target.go

```diff
@@ -0,0 +1,92 @@
+package mg
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+)
+
+// Target checks whether the target files/directories have been more recently
+// modified than src.
+//
+// When recursive is set to true, if src is a directory it will be walked
+// recursively to deermine modified time.
+func Target(src string, recursive bool, targets ...string) (bool, error) {
+	if len(targets) == 0 {
```

Comment:
I would panic here... this is a programmer error... there's no reason to call this with less than one prerequisite.